packages/core/theme/src/components/input.ts

```diff
@@ -69,6 +69,7 @@ const input = tv({
     helperWrapper: "hidden group-data-[has-helper=true]:flex p-1 relative flex-col gap-1.5",
     description: "text-tiny text-foreground-400",
     errorMessage: "text-tiny text-danger",
+    headerWrapper: "flex pb-1 justify-between w-full items-center",
```

Comment:
sorry, could tell me how to update this files or can you change it for me？

Reply:
This change does not seem to affect the docs

Reply:
for the first point, because you added a new slot `headerWrapper`, so you need to document it. See here: https://nextui.org/docs/components/textarea#slots

for the second point, since you added something new to input theme, so you need to update the min peer dependency.